### DISCUSSION TITLE
custom.yaml放进paopaodns里加载不了规则

### DISCUSSION THREAD

**Discussion by jaesony** [kkkgo/PaoPaoGateWay#139] [问和答] [answered]
用docker定制ISO镜像，替换成mihomo核心，如果把ppgw.ini和custom.yaml文件放进去，yacd里可以看到RULE-SET规则。相反如果定制镜像里去除这两个文件，放进paopaodns目录里，就加载不了自定义规则，只加载出proxies字段

**Comment by kkkgo** [comment, marked as answer]
你确认放的是一样的ppgw.ini？猜测你可能是开了fast_node

**Reply by jaesony** [reply]
是开了fast_node，可以了。奇怪的是，确定放进定制docker里没问题

**Reply by kkkgo** [reply]
fast_node会过滤掉所有规则，文档有说明。过滤动作发生在下载配置时。
